gatsby-config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
       resolve: 'gatsby-plugin-favicon',
       options: {
         logo: './src/images/favicon.svg',
+        theme_color: '#C53F78',
```

Comment:
please use the new 'props.data.contentfulBoard.color' property instead of the hardcoded color